cdap-hbase-spi/src/main/java/co/cask/cdap/spi/hbase/HBaseDDLExecutor.java

```diff
@@ -25,18 +25,21 @@
 import javax.annotation.Nullable;
 
 /**
- * Interface providing the HBase DDL operations.
+ * Interface providing the HBase DDL operations. Except the {@link #initialize} method,
```

Comment:
a bit more concise to say 'All methods exception initialize must be idempotent.'